src/algebra/group/prod.lean

```diff
@@ -317,3 +318,16 @@ def prod_units : units (M × N) ≃* units M × units N :=
 end
 
 end mul_equiv
+
+section units
+
+open opposite
+
+/-- Used mainly to define the natural topology of `units α`. -/
```

Comment:
```suggestion
/-- Canonical homomorphism of monoids from `units α` into `α × αᵒᵖ`.
Used mainly to define the natural topology of `units α`. -/
```
Docstrings should lead with a description of the declaration's content.

Reply:
Sure